helmfile.d/values/opensearch/securityadmin.yaml.gotmpl

```diff
@@ -9,6 +9,8 @@ securityConfig:
 
     config:
       dynamic:
+        kibana:
+          server_username: dashboards
```

Comment:
Is this something that they now require being set?
How does the `server_username` relate to opensearch dashboards?

Reply:
So after migrating back to the default index name `.kibana` and getting past the initial System Index permission issues, there were error logs in OpenSearch Dashboards along the lines of "Tenant global_tenant is not allowed for user dashboards" (this is from memory but I could probably reproduce them). So I found this setting which is supposed to match the name of the user that OpenSearch Dashboards will use to authenticate towards OpenSearch (which is `dashboards` in our case), which after setting got rid of this error. But it is a bit unclear to me if this value should be required now but wasn't before, as I didn't find much when searching around for the error.

Reply:
Do we know if `dashboards` is the default user for opensearch-dashboards to use?

Reply:
I can double check but I believe it uses `kibanaserver` by default